src/kv/test/kv_test.cpp

```diff
@@ -2644,4 +2646,179 @@ TEST_CASE("Reported TxID after commit")
     REQUIRE(tx_id->version == store_last_seqno - 1);
     REQUIRE(tx_id.value() == kv_store.current_txid());
   }
-}
\ No newline at end of file
+}
+
+#ifdef KV_STATE_RB
+
+template <typename T>
+std::map<T, T> std_map_range(const std::map<T, T>& map, T from, T to)
+{
+  std::map<T, T> ret;
+  for (auto const& e : map)
+  {
+    if (e.first < from)
+    {
+      continue;
+    }
+    else if (!(e.first < to))
+    {
+      break;
+    }
+
+    ret.emplace(e);
+  }
+
+  return ret;
+}
+
+template <typename T>
+T get_map_get_factor(const std::map<T, T>& map, size_t factor)
+{
+  auto middle_it = map.begin();
+  std::advance(middle_it, map.size() / factor);
+  return middle_it->first;
+}
+
+template <class H, class T>
+std::map<T, T> kv_map_range(H& h, const T& from, const T& to)
+{
+  std::map<T, T> range;
+  auto f = [&range](const T& k, const T& v) { range.emplace(k, v); };
+  h->range(f, from, to);
+  return range;
+}
+
+TEST_CASE("Range")
+{
+  LOG_INFO_FMT("Running KV range tests");
```

Comment:
This is here to check that the test is actually running on the CI, rather than https://github.com/microsoft/CCF/pull/3274#discussion_r760977601.